index.ts

```diff
@@ -116,3 +120,22 @@ console.log(square.getColor());
 console.groupEnd();
 
 console.groupEnd();
+
+const radio = new RadioQuestion(
+  new ConsolePrinter(),
+  ['answ1', 'answ2', 'answ3', 'answ4'],
+  'What?',
+  "I don't know",
+);
+console.group('Radio');
+
+console.log(radio.optionsCount());
+console.log(radio.print());
+console.groupEnd();
+
+const text = new TextQuestion(new ConsolePrinter(), 10, 'What?', 'Nothing');
+console.group('Text');
+
+console.log(text.validation());
+console.log(text.print());
```

Comment:
Don't need to wrap `print` into `console.log`
`print` calls `console.log` inside of itself